beacon_node/store/src/lib.rs

```diff
@@ -157,12 +158,15 @@ pub enum StoreOp<'a, E: EthSpec> {
     PutBlock(Hash256, Arc<SignedBeaconBlock<E>>),
     PutState(Hash256, &'a BeaconState<E>),
     PutBlobs(Hash256, Arc<BlobsSidecar<E>>),
+    PutOrphanedBlobsKey(Hash256),
     PutStateSummary(Hash256, HotStateSummary),
     PutStateTemporaryFlag(Hash256),
     DeleteStateTemporaryFlag(Hash256),
     DeleteBlock(Hash256),
+    DeleteBlobs(Hash256),
     DeleteState(Hash256, Option<Slot>),
     DeleteExecutionPayload(Hash256),
+    PutRawKVStoreOp(KeyValueStoreOp),
```

Comment:
For compatibility with #3931 do you mind renaming this variant to `KeyValueOp`? :pray: Will minimise merge conflicts in future

Reply:
Of course